app/scripts/controllers/transactions.js

```diff
@@ -274,6 +246,22 @@ module.exports = class TransactionController extends EventEmitter {
     })
   }
 
+  cancelTransaction (txId) {
+    this.setTxStatusRejected(txId)
+    return Promise.resolve()
+  }
+
+
+  getChainId () {
+    const networkState = this.networkStore.getState()
+    const getChainId = parseInt(networkState)
```

Comment:
https://github.com/ethereum/wiki/wiki/JSON-RPC#net_version
return value is not a hex